src/app/geschenke/page.tsx

```diff
@@ -15,8 +18,32 @@ export async function generateMetadata() {
   };
 }
 
+const styles = {
+  yearlyOnly: css({
+    display: "block",
+    ":has([value='ABO_GIVE_MONTHS']:checked) &": {
```

Comment:
one thing i'm wondering: is it possible to somehow make this style safer, as in: ensure that `ABO_GIVE_MONTHS` exist? Maybe a const? Maybe also, this should live at the same place as the chooser and then be exported?